### ISSUE TITLE
ios-deploy leaves my terminal in raw mode

### ISSUE THREAD

**Issue by josharian** [phonegap/ios-deploy#96]
I'm running a simple app that just prints hello world to stdout and exits.

After running ios-deploy, my terminal is in raw mode, which is a pain. Anonymized bash output:

```bash
$ ios-deploy -r --debug --bundle hello.app
[snip]
[100%] Connecting to remote debug server
-------------------------
(lldb) command source -s 0 '/tmp/fruitstrap-lldb-prep-cmds-[snip]'
Executing commands in '/private/tmp/fruitstrap-lldb-prep-cmds-[snip]'.
(lldb)     platform select remote-ios --sysroot [snip]
  Platform: remote-ios
 Connected: no
  SDK Path: [snip]
(lldb)     target create [snip]
Current executable set to [snip] (armv7).
(lldb)     script fruitstrap_device_app="/private/var/mobile/Containers/Bundle/Application/[snip]/hello.app"
(lldb)     script fruitstrap_connect_url="connect://127.0.0.1:12345"
(lldb)     command script import "/tmp/fruitstrap_[snip].py"
(lldb)     command script add -f fruitstrap_[snip].connect_command connect
(lldb)     command script add -s asynchronous -f fruitstrap_[snip].run_command run
(lldb)     command script add -s asynchronous -f fruitstrap_[snip].autoexit_command autoexit
(lldb)     command script add -s asynchronous -f fruitstrap_[snip].safequit_command safequit
(lldb)     connect
Process 0 connected
(lldb)     run
success
Process 260 stopped
* thread #1: tid = 0x2212, 0x1fecd000 dyld`_dyld_start, stop reason = shared-library-event
    frame #0: 0x1fecd094 dyld`gdb_image_notifier
dyld`gdb_image_notifier:
-> 0x1fecd094:  bx     lr

dyld`dyldbootstrap::start(macho_header const*, int, char const**, long, macho_header const*, unsigned long*):
   0x1fecd098:  push   {r4, r5, r6, r7, lr}
   0x1fecd09a:  add    r7, sp, #0xc
   0x1fecd09c:  push.w {r8, r10, r11}
Hello, iOS!
Process 260 exited with status = 0 (0x00000000) 
(lldb) Got an empty packet hence killing child (90520) tree
$ $ $ $
```

**Comment by tapasya** [comment]
any updates on this issue ?

**Comment by shazron** [comment]
Try the latest 1.5.0 release, let me know.

**Comment by shazron** [comment]
Closing, should be fixed in 1.5.0
